src/components/BookDetail/BookInformation.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import defaultBookSvg from '../../assets/default_book_image.jpg';
+import './BookInformation.css';
+
+function BookInformation(props) {
+  return (
+    <div className="book-information-container">
+      <img className="book-information-img" src={props.book.image_url || defaultBookSvg} alt="Book" />
+      <div className="book-information-details-container">
+        <div className="book-information-cluster-container">
```

Comment:
this div is unnecesary

Reply:
I don't agree, I use those two divs to group title+author and year+genre as the marketch suggests